.flake8

```diff
@@ -12,10 +13,14 @@ extend-ignore =
     E203,
     # SIM106: Handle error-cases first
     SIM106,
+    ANN101,
```

Comment:
Let's add comments here explaining what each error is.

Reply:
> Minor nits over the lint config 
Added the comments as suggested. Thanks for reviewing :+1: